frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -45,16 +45,16 @@ const plugin: Plugin<ConsumedExtensions> = [
   {
     type: 'FeatureFlag/Model',
     properties: {
-      model: models.CephClusterModel,
-      flag: CEPH_FLAG,
+      model: models.OCSServiceModel,
```

Comment:
just the model as the flag is generic for the ceph extension we don't need to give name corresponding to model we are using.

Reply:
Lets not fix that is not broken